_posts/2022-08-15-requiring-mfa-on-popular-gems.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Requiring MFA on popular gem maintainers
+layout: post
+author: Jenny Shen
+author_email: jenny.shen@shopify.com
+---
+<p align="center">
+    <img src="/images/gem-with-thumbs-up-mfa-dropshadow.png" alt="Doodle of a RubyGem wearing a MFA hat, giving a thumbs up" width="300"/>
+</p>
+
+Two months ago, we outlined our [commitment](https://blog.rubygems.org/2022/06/13/making-packages-more-secure.html) to making Ruby’s supply chain more secure. To combat account takeovers — the second most common software supply chain attack —  we announced a policy to require MFA on at least the top-100 RubyGems packages.
+
+Today (August 15th, 2022), we will begin to enforce MFA on owners of gems with over 180 million total downloads. Users in this category who do not have MFA enabled on the `UI and API` or `UI and gem signin` level will not be able to edit their profile on the web, perform [privileged actions](https://guides.rubygems.org/mfa-requirement-opt-in/#privileged-operations) (i.e. push and yank gems, or add and remove gem owners) or sign in on the command line until they [configure MFA](https://guides.rubygems.org/setting-up-multifactor-authentication/).
```

Comment:
Super tiny nit: It's just a stylistic choice. I prefer the [Oxford comma](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/). Its [proper usage](https://www.grammarly.com/blog/comma-before-or/) would be when there is a list of three or more things, which applies to this case.

That said, with or without the comma, it's correct.

> Users in this category who do not have MFA enabled on the `UI and API` or `UI and gem signin` level will not be able to edit their profile on the web, perform privileged actions (i.e. push and yank gems, or add and remove gem owners),👈  or sign in on the command line until they configure MFA.